src/components/Explore/Explore.tsx

```diff
@@ -63,16 +74,17 @@ function getLabelLength(series: Series, shortLabel: boolean) {
 }
 
 const Explore: React.FunctionComponent<{
-  projection: Projection;
-  chartId?: string;
-}> = ({ projection, chartId }) => {
-  const { locationName, fips } = projection;
+  fips: string;
```

Comment:
The only thing we really needed from the projection was the fips, so I simplified this.

And we don't use chartId anymore, since we grab the sharedComponentId directly below.